src/app/clusters/scenes-server/SceneTable.h

```diff
@@ -186,6 +186,12 @@ class SceneTable
         }
         ~SceneData(){};
 
+        bool operator==(const SceneData & other) const
+        {
+            return ((CharSpan(mName, mNameLength) == CharSpan(other.mName, other.mNameLength)) &&
```

Comment:
CharSpan cannot be directly compared must use `CharSpan(x,y).data_equal(CharSpan(a,b))`

Reply:
Found that out in compile errors:) fixed it.